### PR TITLE
refactor: align delete modals

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
**Make delete entity modal the same format all over**
continuing this ticket: [UI-698](https://linear.app/autokitteh/issue/UI-698/fit-trigger-delete-message-for-the-trigger-type)

The purpose of the task is to create a unified look and feel, to have the same message and the same buttons.
The message should be:
Delete [Entity Name]
Are you sure you want to delete this [entity]? This action cannot be undone, and all related data will be permanently removed. 
[Cancel] [Ok]

Without X close icon on the right top corner.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-720/make-delete-entity-modal-the-same-format-all-over

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/code/deleteModal.tsx [hunk 1]
```diff
@@ -3,45 +3,45 @@ import React from "react";
...
 					{t("cancelButton")}
 				</Button>
 
 				<Button
 					ariaLabel={t("deleteButton")}
-					className="w-auto bg-gray-1100 px-4 py-3 font-semibold"
+					className="bg-gray-1100 px-4 py-3 font-semibold hover:text-error"
 					onClick={onDelete}
 					variant="filled"
 				>
```
Comment (J1za) [inline src/components/organisms/code/deleteModal.tsx:43]: `disabled={isDeleting}` and the same for other modal components

File: src/components/organisms/code/deleteModal.tsx [hunk 2]
```diff
@@ -3,45 +3,45 @@ import React from "react";
...
-					{t("deleteButton")}
+					{isDeleting ? <Loader size="sm" /> : t("deleteButton")}
```
Comment (J1za) [inline src/components/organisms/code/deleteModal.tsx:44]: It is better to leave the text  (`t("deleteButton")}`) and add the `Loader` if isDeleting = true

File: src/components/organisms/code/deleteModal.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/organisms/code/deleteModal.tsx:43]: done

File: src/components/organisms/code/deleteModal.tsx (hunk 2 above)
Comment (RonenMars) [reply src/components/organisms/code/deleteModal.tsx:44]: done

**Review by J1za — APPROVED** [review]

**Pull request merged**